codegen-server/src/main/kotlin/software/amazon/smithy/rust/codegen/server/smithy/generators/CollectionConstraintViolationGenerator.kt

```diff
@@ -96,4 +112,20 @@ class CollectionConstraintViolationGenerator(
             }
         }
     }
+
+    private fun generateDisplayMessageForEachVariant(memberConstraintVariantPresent: Boolean) =
+        writable {
+            collectionConstraintsInfo.forEach {
```

Comment:
nit: not a fan of `forEach` if not already in an FP-style chain. A regular for loop is clearer.